### PR TITLE
Break apart tables for print-friendliness

### PR DESCRIPTION
**Pull request by hbillings**
In #1262, it became clear that this was a larger fix than just "better print styles." This rearranges some of the information to be in multiple stacked tables instead of one long horizontal table. The benefit is that not only will it print better, we can also get rid of some of my janky scrolly table code!

### This pull request changes...
- Breaks some tables (Incentive Payments by FFY, Activity Cost Allocation federal share by quarter) into multiple tables by fiscal year
- Drops the scrolling functionality from two of our larger tables
- Adds some print styles for better scanning on paper

Example of Incentive Payments table:
screen shot 2019-03-06 at 6 14 01 pm

Activity cost allocation:
screen shot 2019-03-08 at 5 47 22 am

### This pull request is ready to merge when...
- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented

### This feature is done when...
- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Comment by jeromeleecms** [comment]
@hbillings 👍 

For consideration + my education: Is it worth breaking out each table change as a separate PR so it's easier to trace back/roll back code if needed (esp. since the specific tables to be updated are still tbd)?

File: web/src/containers/IncentivePayments.js [hunk 1]
```diff
@@ -32,76 +32,28 @@ class IncentivePayments extends Component {
     });
 
     return (
-      <div className="table-frozen-wrapper">
```
Comment (hbillings) [inline web/src/containers/IncentivePayments.js]: we don't need any of the scrolly panel code in these any more; yay!

**Comment by hbillings** [comment]
@mgwalker I've been trying to hunt down why the `IncentivePayments handles changes` test is failing, but I'm really stymied. I suspect it has something to do with the rendering of the component spitting out multiple tables and `ShallowWrapper` not being able to find things within it, but I'm not sure how to go about debugging this.

File: web/src/containers/IncentivePayments.test.js [hunk 2]
```diff
@@ -56,7 +56,6 @@ describe('incentive payments component', () => {
 
     component
       .find('InputHolder')
-      .not('.fake-spacer-input')
```
Comment (hbillings) [inline web/src/containers/IncentivePayments.test.js]: This was something used in the scrolly table code.

File: web/src/styles/scss/print.scss [hunk 3]
```diff
@@ -22,19 +22,24 @@
   }
 
   header,
+  .ds-l-col--3,
```
Comment (hbillings) [inline web/src/styles/scss/print.scss]: Hide the sidebar

File: web/src/styles/scss/print.scss [hunk 4]
```diff
@@ -22,19 +22,24 @@
...
+  .rdw-editor-toolbar,
+  .table-frozen-left-pane {
     display: none;
   }
 
-  .site-main {
-    max-width: 900px;
+
+  .ds-l-container,
+  .ds-l-col--9 {
```
Comment (hbillings) [inline web/src/styles/scss/print.scss]: Override the innate width of these.

File: web/src/styles/scss/print.scss [hunk 5]
```diff
@@ -55,6 +60,19 @@
...
     position: inherit;
   }
+
+  .site-main,
+  .ds-l-container,
+  .accordian > .btn.sm-px3,
+  .has-sticky-header {
+    padding-left: 0;
+    padding-right: 0;
+  }
```
Comment (hbillings) [inline web/src/styles/scss/print.scss]: Get rid of extraneous padding to give us as much page real estate as possible.

File: web/src/styles/scss/print.scss [hunk 6]
```diff
@@ -55,6 +60,19 @@
...
+
+  table input {
+    padding: 0 !important;
+    border: none !important;
```
Comment (hbillings) [inline web/src/styles/scss/print.scss]: There ain't no reason to show input boxes in print!

File: web/src/containers/IncentivePayments.js [hunk 7]
```diff
@@ -30,83 +29,30 @@ class IncentivePayments extends Component {
...
-              ))}
-            </tbody>
-          </table>
-          <table
-            className="table-cms table-fixed table-frozen-data"
-            style={{ minWidth: 1200 }}
-          >
+      <Fragment>
+        {years.map(( year ) => (
+        <Fragment>
```
Comment (hbillings) [inline web/src/containers/IncentivePayments.js]: I've got a bunch of nested `Fragment`s here; I'm not sure if that's the best practice. (I also suspect maybe it's what's throwing off the test?)

File: web/src/containers/IncentivePayments.js [hunk 8]
```diff
@@ -121,42 +67,35 @@ class IncentivePayments extends Component {
...
-                      headers={tdHdrs('total', 'grand')}
-                    >
-                      {formatNumber(type, totals[id].allYears)}
-                    </td>
+                      ))}
+                      <td
+                        className="bold mono right-align align-middle"
+                        headers={tdHdrs(year, 'subtotal')}
+                      >
+                        {formatNumber(totals[id].byYear[year])}
```
Comment (mgwalker) [inline web/src/containers/IncentivePayments.js]:
```suggestion
                        {formatNumber(type, totals[id].byYear[year])}
```

The subtotals column is ending up always as `--` - `formatNumber` needs the number type as its first argument

**Pull request merged**